security/security_profiles_operator/spo-seccomp.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="spo-seccomp"]
+= Managing seccomp profiles
+include::_attributes/common-attributes.adoc[]
+:context: spo-seccomp
+
+toc::[]
+
+Create and manage seccomp profiles and bind them to workloads.
+
+include::modules/spo-creating-profiles.adoc[leveloffset=+1]
+
+include::modules/spo-applying-profiles.adoc[leveloffset=+1]
+
+include::modules/spo-binding-workloads.adoc[leveloffset=+2]
+
+include::modules/spo-recording-profiles.adoc[leveloffset=+1]
+
+include::modules/spo-container-profile-instances.adoc[leveloffset=+2]
+
+[discrete]
+[role="_additional-resources"]
+== Additional resources
```

Comment:
The additional resources headings in this PR [should have IDs](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies), even when they are [discrete](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#discrete-headings).